.github/workflows/build_centos7.yml

```diff
@@ -163,38 +167,165 @@ jobs:
+        if: always()
+        id: save-cache-vcpkg-binary
+        uses: actions/cache/save@v3
+        with:
+          path: ${{ github.workspace }}/vcpkg_cache
+          key: vcpkg-cache-ubuntu-${{ hashFiles('src/vcpkg.json', '.git/modules/vcpkg/HEAD') }}
+
       - name: Running unit tests
         timeout-minutes: 120
         shell: bash
         run: |
-          export PATH=${GITHUB_WORKSPACE}/vcpkg_installed/x64-linux/tools/openmpi/bin/:$PATH
+          export PATH=${GITHUB_WORKSPACE}/_build/vcpkg_installed/x64-linux/tools/openmpi/bin/:$PATH
           export LD_LIBRARY_PATH=LD_LIBRARY_PATH:${{ env.XPRS_LIB_Path_CONTAINER }}
           export XPRESS=${{ env.XPRESS_CONTAINER }}
           cd _build
           ctest -C Release --output-on-failure -L "unit|benders|lpnamer|medium"
+
+      #######################
+      - name: Install
+        run: |
+          cd _build
+          cmake --install .
+
+      - name: set name variables
+        id: single_file_name
+        shell: bash
+        run: |
+          VERSION=${{steps.antares-xpansion-version.outputs.result}}
+          echo "VERSION=$VERSION" >> $GITHUB_ENV
+
+      - id: create-single-file
+        name: Single file .tar.gz creation
+        uses: ./.github/workflows/single-file-creation-tgz
+        with:
+          antares-xpansion-version: ${{env.VERSION}}
+
+      - name: Installer .tar.gz creation
+        run: |
+          cd _build
+          export FILE_NAME="antaresXpansion-${{env.VERSION}}-CentOS-7.9.2009"
+          cpack -G TGZ -D CPACK_PACKAGE_FILE_NAME=$FILE_NAME
+          #Need to differentiate between xpress/no_xpress files
+          #Cpack command line doesn't seem to care about -P or -R options
+          echo "TGZ_NAME=$FILE_NAME.tar.gz" >> $GITHUB_ENV
+
+      - name: Upload .tar.gz
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{env.TGZ_NAME}}
+          path: _build/${{env.TGZ_NAME}}
+
+      - name: Upload single file
+        uses: actions/upload-artifact@v3
+        with:
+          name: ${{ steps.create-single-file.outputs.archive-name }}
+          path: ${{ steps.create-single-file.outputs.archive-path }}
+
+      - name: export asset name in output
+        id: export_output
+        run: |
+          echo "TGZ_NAME=${{env.TGZ_NAME}}" >> $GITHUB_OUTPUT
+          echo "SINGLE_FILE_NAME=${{ steps.create-single-file.outputs.archive-name }}" >> $GITHUB_OUTPUT
+
+      #######################
+  userguide:
+    if: github.event_name == 'release' && github.event.action == 'created'
+    runs-on: ubuntu-latest
+    outputs:
+      pdf-name: ${{ steps.create-user-guide.outputs.pdf-name }}
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Read antares-xpansion version
+        id: antares-xpansion-version
+        uses: ./.github/actions/read-json-value
+        with:
+          path: 'antares-version.json'
+          key: 'antares_xpansion_version'
+
+      - id: create-user-guide
```

Comment:
repeated on every platform

Reply:
Yes unfortunatly. May be the object of another improvement